src/components/billing/info/infobar.styles.ts

```diff
@@ -0,0 +1,71 @@
+import { makeStyles } from "@mui/styles";
+
+const InfoBarStyles = makeStyles({
+  container: {
+    position: "relative",
+    display: "flex",
+    width: "928px",
+    height: "150px",
```

Comment:
We discussed earlier that we will not use fixed width and height and let the content drive the width